### ISSUE TITLE
Doze issues?

### ISSUE THREAD

**Issue by shellstrom** [liato/android-bankdroid#577]
Ever since I got Marshmallow on my N6 I very frequently have to re sign in to allow for widgets to update (all the widgets get exclamation marks on them). Pre-Marshmallow this seemed to work more or less flawlessly.
I suspect the problems I'm seeing is a result of the newly introduced Doze feature which Bankdroid hasn't been adapted to accommodate for yet.

**Comment by fredrike** [comment]
In API 23 the notifications needs to be re-written..

Something like this:

```
@@ -122,7 +123,13 @@ public class AutoRefreshService extends Service {
         final PendingIntent contentIntent = PendingIntent.getActivity(context, 0,
                 new Intent(context, MainActivity.class), 0);
 
-        notification.setLatestEventInfo(context, bank.getDisplayName(), text, contentIntent);
+        NotificationCompat.Builder nB = new NotificationCompat.Builder(context);
+        nB.setContentText(text);
+        nB.setContentTitle(bank.getDisplayName());
+        nB.setContentIntent(contentIntent);
+
```

**Comment by goober** [comment]
@fredrike Do you know the status on this one? Did you do the necessary updates?

**Comment by fredrike** [comment]
Closed by cc2647dfddc6e57d294186f3a05b30a21cb1b992
